### PR TITLE
feat: add `kili.delete_labels()`

### PR DESCRIPTION
**Pull request by Jonas1312**
to merge when the MR on kili staging is merged

### CODE REVIEW

File: src/kili/entrypoints/mutations/label/__init__.py [hunk 1]
```diff
@@ -325,3 +326,17 @@ def create_honeypot(
...
+    def delete_labels(self, ids: List[str]):
+        """Delete labels.
+
+        Args:
+            ids: List of label ids to delete.
+
+        Returns:
+            The deleted label ids.
+        """
+        variables = {"ids": ids}
```
Comment (PierreLeveau) [inline src/kili/entrypoints/mutations/label/__init__.py:340]: we should batch the call for a number of labels > 100

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

**Review by theodu — APPROVED** [review]

File: tests/integration/entrypoints/mutations/test_labels.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+from kili.entrypoints.mutations.label import GQL_DELETE_LABELS, MutationsLabel
+
+
+def test_delete_labels(mocker: pytest_mock.MockerFixture):
+    kili = MutationsLabel()
+    kili.graphql_client = mocker.MagicMock()
+    kili.http_client = mocker.MagicMock()
+
+    kili.delete_labels(ids=["id1", "id2"])
+
```
Comment (PierreLeveau) [inline tests/integration/entrypoints/mutations/test_labels.py:12]: i would put 101 labels and check that the client has been called twice

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

**Comment by theodu** [comment]
```kili.delete_labels(ids=[])

GraphQLError: GraphQL error at index 0: WHERE parameter "id" has invalid "undefined" value"
```

we might need to do an early return if we have an empty list

**Review by theodu — CHANGES_REQUESTED** [review]
```kili.delete_labels(ids=[])
GraphQLError: GraphQL error at index 0: WHERE parameter "id" has invalid "undefined" value"
```

we might need to do an early return if we have an empty list

**Review by theodu — APPROVED** [review]

**Pull request merged**